[2025-02-25T18:00:01.719+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-25T18:00:01.758+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tugas_kita.wait_for_new_data manual__2025-02-25T17:56:39.401965+00:00 [queued]>
[2025-02-25T18:00:01.805+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tugas_kita.wait_for_new_data manual__2025-02-25T17:56:39.401965+00:00 [queued]>
[2025-02-25T18:00:01.811+0000] {taskinstance.py:2866} INFO - Starting attempt 4 of 6
[2025-02-25T18:00:01.919+0000] {taskinstance.py:2889} INFO - Executing <Task(SqlSensor): wait_for_new_data> on 2025-02-25 17:56:39.401965+00:00
[2025-02-25T18:00:01.968+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=29680) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-25T18:00:01.973+0000] {standard_task_runner.py:72} INFO - Started process 29712 to run task
[2025-02-25T18:00:01.991+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'tugas_kita', 'wait_for_new_data', 'manual__2025-02-25T17:56:39.401965+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/tugas_akhir.py', '--cfg-path', '/tmp/tmpj7orftaj']
[2025-02-25T18:00:01.998+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask wait_for_new_data
[2025-02-25T18:00:02.177+0000] {task_command.py:467} INFO - Running <TaskInstance: tugas_kita.wait_for_new_data manual__2025-02-25T17:56:39.401965+00:00 [running]> on host d22a487e6b55
[2025-02-25T18:00:02.406+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Dion Rikki' AIRFLOW_CTX_DAG_ID='tugas_kita' AIRFLOW_CTX_TASK_ID='wait_for_new_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-25T17:56:39.401965+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-25T17:56:39.401965+00:00'
[2025-02-25T18:00:02.411+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-25T18:00:02.413+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-25T18:00:02.415+0000] {logging_mixin.py:190} INFO - Current task name:wait_for_new_data state:running start_date:2025-02-25 18:00:01.761932+00:00
[2025-02-25T18:00:02.419+0000] {logging_mixin.py:190} INFO - Dag name:tugas_kita and current dag run status:running
[2025-02-25T18:00:02.422+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-25T18:00:02.467+0000] {base.py:84} INFO - Retrieving connection 'mysql_default'
[2025-02-25T18:00:02.477+0000] {sql.py:101} INFO - Poking: SELECT COUNT(*) FROM tugas_akhir_sensordata WHERE timestamp >= NOW() - INTERVAL 5 MINUTE (with parameters None)
[2025-02-25T18:00:02.490+0000] {base.py:84} INFO - Retrieving connection 'mysql_default'
[2025-02-25T18:00:02.503+0000] {sql.py:553} INFO - Running statement: SELECT COUNT(*) FROM tugas_akhir_sensordata WHERE timestamp >= NOW() - INTERVAL 5 MINUTE, parameters: None
[2025-02-25T18:00:02.507+0000] {sql.py:562} INFO - Rows affected: 1
[2025-02-25T18:00:02.511+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-02-25T18:00:02.532+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-25T18:00:02.534+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=tugas_kita, task_id=wait_for_new_data, run_id=manual__2025-02-25T17:56:39.401965+00:00, execution_date=20250225T175639, start_date=20250225T180001, end_date=20250225T180002
[2025-02-25T18:00:02.580+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-25T18:00:02.581+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-25T18:00:02.583+0000] {logging_mixin.py:190} INFO - Dag name:tugas_kita queued_at:2025-02-25 17:56:39.425866+00:00
[2025-02-25T18:00:02.584+0000] {logging_mixin.py:190} INFO - Task hostname:d22a487e6b55 operator:SqlSensor
[2025-02-25T18:00:02.646+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-25T18:00:02.787+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-25T18:00:02.800+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
